src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -738,6 +738,29 @@ add_subset_sensitivity(TargetT& sensitivity, const int subset_num) const
   const int min_segment_num = -this->max_segment_num_to_process;
   const int max_segment_num = this->max_segment_num_to_process;
 
+#ifdef STIR_MPI
```

Comment:
ideally we use the new `distributable` function always, i.e. get rid of the old code (just put it in `#if 1 ... #endif`). This will also mean support for OPENMP (and easier debugging)